### ISSUE TITLE
How can I resize the batchsize to 2 or 4

### ISSUE THREAD

**Issue by WHY-student** [franciszzj/HiLo#8]
Thank you for your code, which is helpful for me.
However, when I resize the batchsize to 2 or 4 during train process, there is an error happened in loss function

The operation
```.python
                        high2low_s_cls_mask = F.one_hot(high2low_idx2.reshape([num_dec_layers, 1]), num_classes=self.num_queries).unsqueeze(-1).expand(-1, -1, -1, high2low_all_s_cls_scores.shape[-1]).bool()  # noqa
                        this_high2low_s_cls = torch.masked_select(high2low_all_s_cls_scores, high2low_s_cls_mask).reshape((num_dec_layers, -1)).softmax(-1)  # noqa
                        high2low_o_cls_mask = F.one_hot(high2low_idx2.reshape([num_dec_layers, 1]), num_classes=self.num_queries).unsqueeze(-1).expand(-1, -1, -1, high2low_all_o_cls_scores.shape[-1]).bool()  # noqa
                        this_high2low_o_cls = torch.masked_select(high2low_all_o_cls_scores, high2low_o_cls_mask).reshape((num_dec_layers, -1)).softmax(-1)  # noqa
                        high2low_s_bbox_mask = F.one_hot(high2low_idx2.reshape([num_dec_layers, 1]), num_classes=self.num_queries).unsqueeze(-1).expand(-1, -1, -1, high2low_all_s_bbox_preds.shape[-1]).bool()  # noqa
```
is prevented by the inequality between high2low_idx2.shape (`[18]`, when  I use batchsize=2) and` [num_dec_layers, 1]([9,1])`.

I guessed that the 1 in [num_dec_layers, 1] should be the batchsize?
Could you help me to solve the problem so that I can resize the batchsize to 2 or 4?

**Comment by franciszzj** [comment]
The current code only supports a batch size of 1 for each GPU.

**Comment by franciszzj** [comment]
Although some HiLo code does not support larger batch_size, **HiLo Baseline** does support it and you can try it.

**Comment by WHY-student** [comment]
Thank you for your reply， I will try it
